Dockerfile

```diff
@@ -20,18 +21,20 @@ RUN cmake CMakeLists.txt && make && cp ./*.a /usr/lib
 
 ADD . /opt/OpenDDS
 
+ARG ACE_CONFIG_OPTION="--doc-group"
 RUN cd /opt/OpenDDS && \
-    ./configure --prefix=/usr/local --security --doc-group --std=c++11 && \
+    ./configure --prefix=/usr/local --security --std=c++11 ${ACE_CONFIG_OPTION} && \
     make && \
     make install && \
-    cp -a /opt/OpenDDS/ACE_wrappers/MPC /usr/local/share/ace/MPC
+    . /opt/OpenDDS/setenv.sh && \
+    cp -a ${MPC_ROOT} /usr/local/share/MPC
 
 ENV ACE_ROOT=/usr/local/share/ace \
     TAO_ROOT=/usr/local/share/tao \
     DDS_ROOT=/usr/local/share/dds \
     PATH=".:/usr/local/share/ace/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 
 WORKDIR /opt/OpenDDS/tests/DCPS/Messenger
-RUN mwc.pl -type gnuace && make
+RUN . /opt/OpenDDS/setenv.sh && mwc.pl -type gnuace && make
```

Comment:
Does it work without this setenv?  Shouldn't be needed due to the docker `ENV` above.

Reply:
I can't remember the specific error but it was looking for the MPC_ROOT so I used the setenv.sh in this run instruction.

Reply:
Just reran. Here are the details:
Step 13/14 : RUN mwc.pl -type gnuace && make
 ---> Running in 168179d6a35e
ERROR: Unable to find the MPC modules in /usr/local/share/ace/MPC.
You can set the MPC_ROOT environment variable to the location of MPC.
The command '/bin/sh -c mwc.pl -type gnuace && make' returned a non-zero code: 255

Reply:
Yes, it needs MPC_ROOT, but you shouldn't use setenv.sh here since that's the source tree and not the installed tree.  You can update the ENV block above to have the correct MPC_ROOT